cdap-common/src/test/java/co/cask/cdap/common/test/AppJarHelper.java

```diff
@@ -102,7 +102,7 @@ public static Location createDeploymentJar(LocationFactory locationFactory, Clas
           }
 
           // TODO: this is due to manifest possibly already existing in the jar, but we also
-          // create a manifest programatically so it's possible to have a duplicate entry here
+          // create a manifest programmatically so it's possible to have a duplicate entry here
```

Comment:
haha... this class is copied from Hadoop... and the `PluginJarHelper` was copied from this class.....